layout/loadGeoTerrain.m

```diff
@@ -0,0 +1,197 @@
+function [terrain, trajectories] = loadGeoTerrain(Monster, numUsers)
```

Comment:
Maybe refactor this into the mobility class? Thus Mobility class is an abstract class consisting of base properties (i.e. velocity and trajectory)